core/src/main/scala/com/salesforce/op/dsl/RichTextFeature.scala

```diff
@@ -366,16 +370,18 @@ trait RichTextFeature {
      *                       In this case, the tokens will be equivalent to the output from (without empty tokens).
      * @param minTokenLength minimum token length, >= 1.
      * @param toLowercase    indicates whether to convert all characters to lowercase before analyzing
+     * @param stripHtml      indicates whether to strip HTML tags from the text or not before analyzing
      * @return tokenized feature
      */
     def tokenizeRegex
     (
       pattern: String,
       group: Int = -1,
       minTokenLength: Int = TextTokenizer.MinTokenLength,
-      toLowercase: Boolean = TextTokenizer.ToLowercase
+      toLowercase: Boolean = TextTokenizer.ToLowercase,
+      stripHtml: Boolean = TextTokenizer.StripHtml
     ): FeatureLike[TextList] = {
-
+      // html stripping won't work here due since LuceneRegexTextAnalyzer
```

Comment:
if HTML stripping will not work then please dont put it as an input

Reply:
done !